src/controls/checkboxQuickPick.ts

```diff
@@ -4,96 +4,95 @@
 
 import vscode = require("vscode");
 
-var confirmItemLabel: string = "$(checklist) Confirm";
-var checkedPrefix: string = "[ $(check) ]";
-var uncheckedPrefix: string = "[     ]";
-var defaultPlaceHolder: string = "Select 'Confirm' to confirm or press 'Esc' key to cancel";
+const confirmItemLabel: string = "$(checklist) Confirm";
+const checkedPrefix: string = "[ $(check) ]";
+const uncheckedPrefix: string = "[     ]";
+const defaultPlaceHolder: string = "Select 'Confirm' to confirm or press 'Esc' key to cancel";
 
-export interface CheckboxQuickPickItem {
+export interface ICheckboxQuickPickItem {
     label: string;
     description?: string;
     isSelected: boolean;
 }
 
-export interface CheckboxQuickPickOptions {
+export interface ICheckboxQuickPickOptions {
     confirmPlaceHolder: string;
 }
 
-var defaultOptions:CheckboxQuickPickOptions = { confirmPlaceHolder: defaultPlaceHolder};
+const defaultOptions: ICheckboxQuickPickOptions = { confirmPlaceHolder: defaultPlaceHolder};
 
 export function showCheckboxQuickPick(
-    items: CheckboxQuickPickItem[],
-    options: CheckboxQuickPickOptions = defaultOptions): Thenable<CheckboxQuickPickItem[]> {
+    items: ICheckboxQuickPickItem[],
+    options: ICheckboxQuickPickOptions = defaultOptions): Thenable<ICheckboxQuickPickItem[]> {
 
     return showInner(items, options).then(
         (selectedItem) => {
             // We're mutating the original item list so just return it for now.
             // If 'selectedItem' is undefined it means the user cancelled the
             // inner showQuickPick UI so pass the undefined along.
-            return selectedItem != undefined ? items : undefined;
-        })
+            return selectedItem !== undefined ? items : undefined;
+        });
 }
 
-function getQuickPickItems(items: CheckboxQuickPickItem[]): vscode.QuickPickItem[] {
+function getQuickPickItems(items: ICheckboxQuickPickItem[]): vscode.QuickPickItem[] {
 
-    let quickPickItems: vscode.QuickPickItem[] = [];
+    const quickPickItems: vscode.QuickPickItem[] = [];
     quickPickItems.push({ label: confirmItemLabel, description: "" });
 
-    items.forEach(item =>
+    items.forEach((item) =>
         quickPickItems.push({
                 label: convertToCheckBox(item),
-                description: item.description
+                description: item.description,
```

Comment:
this extra `,` was always weird to me... but I'm not picky

Reply:
Trailing comma rules seem to be pretty common in the JavaScript linter world...

Reply:
for sure -  just commenting that it looks weird to me. Especially since in JSON it yells at you if you have a trailing comma.

Don't change because it looks weird to me though :) Let's do what the linters say.